code/game/objects/items/toy_mechs.dm

```diff
@@ -263,12 +264,8 @@
 	if(wins || losses)
 		. += span_notice("This toy has [wins] wins, and [losses] losses.")
 
-/**
- * Override the say proc if they're mute
- */
-/obj/item/toy/mecha/say(message, bubble_type, list/spans = list(), sanitize = TRUE, datum/language/language = null, ignore_spam = FALSE, forced = null, filterproof = null, message_range = 7, datum/saymode/saymode = null)
-	if(!quiet)
-		. = ..()
+/obj/item/toy/mecha/can_speak(allow_mimes)
+	return ..() && !quiet
```

Comment:
```suggestion
	return !quiet && ..()
```

shortcircuiting so we don't have to waste time on a useless parent call (unless parent _needs_ to be invoked for some reason)